src/v/pandaproxy/api/api-doc/schema_registry.json

```diff
@@ -221,6 +221,13 @@
             "schema": {
               "$ref": "#/definitions/mode"
             }
+          },
+          {
+            "name": "force",
+            "description": "Whether to force setting mode to IMPORT when there are existing schemas",
```

Comment:
```suggestion
            "description": "Whether to force setting the mode to IMPORT even when a schema already exists.",
```
Let me know if this changes the meaning too much. I also had a question regarding "override" in the other PR